requirements-dev.txt

```diff
@@ -9,7 +9,7 @@ flake8-import-order
 flake8-mutable
 flake8-quotes
 geographiclib
-# geopandas
```

Comment:
@Conengmo this is a bug in micromamba and I reported it in https://github.com/mamba-org/mamba/issues/579

Pinging @wolfv for awereness. Everything works fine without `geopandas`, if I add `geopandas` micromamba wants to downgrade the Python versions.